sql-plugin/src/main/scala/com/nvidia/spark/rapids/HiveProvider.scala

```diff
@@ -18,7 +18,7 @@ package com.nvidia.spark.rapids
 
 import org.apache.spark.sql.catalyst.expressions.Expression
```

Comment:
Add 2023 to Copyright

Reply:
Also, check other files

Reply:
Thank you, I will re check all of them again